### DISCUSSION TITLE
Question: how to set process title for ```torch.multiprocessing.spawn```

### DISCUSSION THREAD

**Discussion by ubless607** [dvarrazzo/py-setproctitle#136] [General]
Thanks for the nice extension!

Currently, I am training deep learning model with [pytorch DDP](https://pytorch.org/tutorials/intermediate/ddp_tutorial.html).

`torch.multiprocessing.spawn(run, args=(exp,), nprocs=exp.world_size)` creates subprocesses inside.
How can I set user-defined process name for these kind of multiprocessing stuff?

Reference
https://pytorch.org/docs/stable/_modules/torch/multiprocessing/spawn.html#spawn

**Comment by dvarrazzo** [comment]
I assume by calling `setproctitle()` from the `run` function.
